crates/matrix-sdk/src/room/identity_status_changes.rs

```diff
@@ -221,6 +227,27 @@ mod tests {
         assert_eq!(change.len(), 1);
     }
 
+    #[async_test]
+    async fn test_when_user_becomes_verification_violation_we_report_it() {
+        // Given a room containing us and Bob
+        let t = TestSetup::new_room_with_other_member().await;
+
+        // And Bob's identity is verified
+        t.verify().await;
```

Comment:
Would it help if the methods were called e.g. `verify_bob`? The reason I wrote the test helper is because I like my tests to read like a simple Given, When, Then that preferably fits on the screen, and without it I had pages of code for each step.

I think you and I have two slightly different styles here - I often see tests from you that follow a longer narrative, and I see the benefits of that approach, but I am really sold on the idea that if I introduce a bug, one test will fail that very directly tests specifically for that one problem, and nothing else, so I can immediately see what the problem is.

I also really struggle to understand code that is longer than one screen :-)

So the test helper is _intended_ to make it easier to understand, by naming the various actions we are taking and moving the details away somewhere else to read if you want to.

Where it falls down is if you can't guess what the helpful functions do from their name :-(

Reply:
And I want to thank you for your reviews, which are always thoughtful and take a lot of hard work.

Yeah, while I was writing this stuff I was uncomfortable with the method names but couldn't think how to do it better. I think you've put your finger on the problem - it's not clear what data is changing and what transitions are happening.

I will try some renames and ask you for another quick look if that's OK.

Reply:
Renamed in c43ef744db . I think this is definitely better, thanks!